src/cljs/rems/user_settings.cljs

```diff
@@ -80,10 +80,30 @@
                    (when first-time? ; to avoid an infinite loop if the settings fail to save (e.g. unsupported language in cookies)
                      [[:check-if-should-save-language!]]))})))
 
+(rf/reg-event-fx
+ :flash-message-error
```

Comment:
now that this event is just a oneliner, I'd rather get rid of it (and inline it into `:loaded-user-settings-fail`)